apps/api/app/workers/tasks/workflow_tasks.py

```
@@ -255,18 +266,86 @@ async def execute_workflow_by_id(
+                    if reset_time_str:
+                        # Quota exhausted — show when the limit resets
+                        try:
+                            reset_dt = datetime.fromisoformat(reset_time_str)
+                            formatted_reset = reset_dt.strftime("%b %d at %I:%M %p UTC")
+                            body = (
+                                f"'{workflow.title}' couldn't run — "
+                                f"you've used all your workflow executions for today. "
+                                f"Resets {formatted_reset}. "
+                                f"Upgrade to {plan_required} for higher daily limits."
+                            )
+                        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)